From f804ff0864c65caaf8a3356357024639aa785911 Mon Sep 17 00:00:00 2001
From: Sebastian Wagner <sebix@sebix.at>
Date: Sat, 8 Aug 2020 10:07:07 +0200
Subject: [PATCH] fix setup.cfg in flow_dsl project to include data

copy and paste error causes necessary data to be not included in builds
---
 CHANGELOG.md                                              | 2 ++
 tests/functional/registration/projects/flow_dsl/setup.cfg | 4 ++--
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/tests/functional/registration/projects/flow_dsl/setup.cfg b/tests/functional/registration/projects/flow_dsl/setup.cfg
index e9e6f366..aec061de 100644
--- a/tests/functional/registration/projects/flow_dsl/setup.cfg
+++ b/tests/functional/registration/projects/flow_dsl/setup.cfg
@@ -8,8 +8,8 @@ install_requires =
     textX
 
 [options.package_data]
-data_dsl = *.tx
+flow_dsl = *.tx
 
 [options.entry_points]
 textx_languages =
-    flow_dsl = flow_dsl:flow_dsl 
\ No newline at end of file
+    flow_dsl = flow_dsl:flow_dsl
